pkg/sql/gcjob_test/gc_job_test.go

```diff
@@ -311,12 +311,21 @@ func TestGCJobRetry(t *testing.T) {
 	tdb.Exec(t, "CREATE TABLE foo (i INT PRIMARY KEY)")
 	tdb.Exec(t, "INSERT INTO foo SELECT generate_series(1, 1000)")
 	tdb.Exec(t, "DROP TABLE foo")
-	var jobID string
+
+	var jobID, status, lastRun, nextRun, numRuns string
 	tdb.QueryRow(t, `
-SELECT job_id
+SELECT job_id, status, last_run, next_run, num_runs
```

Comment:
hm, i don't think this is where we want to query this info. the `CheckQueryResultsRetry` function is a wrapper around a loop, so that query will run multiple times for up to 45 seconds. 

but in order to query and log this info from within the retry loop, we may need to change it to use `testutils.SucceedsSoon`